actors/miner/src/policy.rs

```diff
@@ -119,6 +119,25 @@ pub fn quality_for_weight(
         .div_floor(&QUALITY_BASE_MULTIPLIER)
 }
 
+// Simple QA power calculation.
+// All values are scaled by the same time so only a space average is taken
+pub fn quality_for_weight_simple(
+    size: SectorSize,
+    deal_weight: &DealWeight,
+    verified_weight: &DealWeight,
+) -> SectorQuality {
+    let sector_space = BigInt::from(size as u64);
+    let total_deal_space = deal_weight + verified_weight;
```

Comment:
These values are still space_time.

I might be misunderstanding, but the sector_space is *not* scaled by the same duration, so I don't think the calculation is correct.

Reply:
No the idea is that these values are now just space, deal weight just means space in the simple context, and this is why im not multiplying by duration [here](https://github.com/filecoin-project/builtin-actors/pull/637/files#r969055437).  This confusion of what dealweight means is inherent in having sector info's fields mean different things which in part is why I prefer the other approach.  From your PR comment it sounds like you are ok with that so I will revert the last commit and see what you think.

Reply:
Ok. FWIW if we did it the "right" way, I would not use the name or type "weight" for things that are not multiplied by duration (except unavoidably in the sector info)